js/src/sdk/client/services.gen.ts

```diff
@@ -318,6 +318,17 @@ export class TriggersService {
         });
     }
     
+    /**
+     * Update new webhook
+     * Update isNewWebhook
+     */
+    public static updateNewWebhook<ThrowOnError extends boolean = false>(options?: Options<UpdateNewWebhookData, ThrowOnError>) {
```

Comment:
The `updateNewWebhook` method lacks proper error handling and validation. Consider adding input validation and documenting possible error scenarios in the method's JSDoc.